src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java

```diff
@@ -1310,7 +1428,7 @@ private void visitClassType(Type type, ReviewLine reviewLine, TokenKind kind) {
         SpacingState spacingBefore = spacingState;
 
         if (type.isPrimitiveType()) {
-            reviewLine.addToken(kind, type.asPrimitiveType().toString(), spacingState.getSpacing());
+            reviewLine.addToken(kind, type.asPrimitiveType().getType().asString(), spacingState.getSpacing());
```

Comment:
Same here - we should ensure the json output doesn't change

Reply:
Same case as above.